templates/tab/js/dashboard/src/App.jsx

```diff
@@ -1,60 +1,52 @@
+import "./App.css";
+
 import { HashRouter as Router, Redirect, Route } from "react-router-dom";
 
 // https://fluentsite.z22.web.core.windows.net/quick-start
 import {
   FluentProvider,
-  Spinner,
-  teamsLightTheme,
   teamsDarkTheme,
   teamsHighContrastTheme,
-  tokens,
+  teamsLightTheme,
 } from "@fluentui/react-components";
-import { useTeamsUserCredential } from "@microsoft/teamsfx-react";
+import { useTeams } from "@microsoft/teamsfx-react";
 
+import { TeamsFxContext } from "./internal/context";
 import SampleDashboard from "./views/dashboards/SampleDashboard";
 import Privacy from "./views/Privacy";
 import TabConfig from "./views/TabConfig";
 import TermsOfUse from "./views/TermsOfUse";
-import { TeamsFxContext } from "./internal/context";
 
 /**
  * The main app which handles the initialization and routing
  * of the app.
  */
 export default function App() {
-  const { loading, themeString, teamsUserCredential } = useTeamsUserCredential({
-    initiateLoginEndpoint: process.env.REACT_APP_START_LOGIN_PAGE_URL,
-    clientId: process.env.REACT_APP_CLIENT_ID,
-  });
+  const { themeString } = useTeams({})[0];
   return (
-    <TeamsFxContext.Provider value={{ themeString, teamsUserCredential }}>
+    <TeamsFxContext.Provider value={{ themeString }}>
       <FluentProvider
+        id="fluent-provider"
         theme={
           themeString === "dark"
             ? teamsDarkTheme
             : themeString === "contrast"
             ? teamsHighContrastTheme
             : teamsLightTheme
         }
-        style={{
-          height: "100vh",
-          background: tokens.colorNeutralBackground3,
-        }}
       >
         <Router>
           <Route exact path="/">
             <Redirect to="/tab" />
           </Route>
-          {loading ? (
-            <Spinner style={{ margin: 100 }} />
-          ) : (
+          {
```

Comment:
Why do we remove the loading indicator?

Reply:
This was removed by mistake, it has been re-added, thanks for helping to find this problem.